torchfix/torchfix.py

```diff
@@ -25,18 +25,55 @@
 
 DISABLED_BY_DEFAULT = ["TOR3", "TOR4"]
 
+ALL_VISITOR_CLS = [
+    TorchDeprecatedSymbolsVisitor,
+    TorchRequireGradVisitor,
+    TorchSynchronizedDataLoaderVisitor,
+    TorchVisionDeprecatedPretrainedVisitor,
+    TorchVisionDeprecatedToTensorVisitor,
+    TorchUnsafeLoadVisitor,
+    TorchReentrantCheckpointVisitor,
+]
+
+_ALL_ERROR_CODES = None
+
+def GET_ALL_ERROR_CODES():
+    global _ALL_ERROR_CODES
```

Comment:
Here `_ALL_ERROR_CODES` just to cache and don't recompute?
Maybe just use `@functools.cache` then?
(It's ok to refactor in later in a different PR).

Reply:
Good point, will update here